cypress/integration/Train/ScenarioAndTags.spec.js

```diff
@@ -0,0 +1,149 @@
+const models = require('../../support/Models')
```

Comment:
fyi, i'm pretty certain another reviewer has switch incoming test code to use imports and will make a similar note in his code review

Reply:
Thanks, I fixed it.